src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/ObjectExplorerService.cs

```diff
@@ -123,11 +123,11 @@ public override void InitializeService(IProtocolEndpoint serviceHost)
 
             // Not enabling parallel processing for OEService as it might cause OE to hang.

```

Comment:
Remove this, and have you verified that whatever was causing this issue is definitely not going to be a problem anymore?

Reply:
I haven't noticed any hangs with this and there would have been other reasons why it happened before, as we noticed it happening even with disabled PMP - will remove comment.